src/OrchardCore.Modules/OrchardCore.Admin/Views/AdminDashboard.cshtml

```diff
@@ -1,4 +1,5 @@
+@inject OrchardCore.IOrchardHelper OrchardHelper
 <div class="alert alert-success">
     <h4>@T["Welcome to Orchard"]</h4>
-    <p>@T["Feel free to browse the menu and discover all its possibilities."]</p>
+    <p>@T[$"Feel free to browse the menu on the {(OrchardHelper.CultureDir() == "ltr" ? "left" : "right")} and discover all its possibilities."]</p>
```

Comment:
It's worse than anything. A translated string can't be dynamic, we can't find the values automatically. And even if in this case there are only two possibilities, it's bad. Either remove the term as you did earlier, or leave it as it was initially and we can handle it in a localization entry.

Reply:
So that's why I remove it at the beginning :)